test/e2e/framework.go

```diff
@@ -2752,7 +2752,10 @@ func (data *TestData) collectAntctlCovFilesFromControlPlaneNode(covDir string) e
 	// copy antctl coverage files from node to the coverage directory
 	var cmd string
 	if testOptions.providerName == "kind" {
-		cmd = fmt.Sprintf("/bin/sh -c find %s -maxdepth 1 -name 'antctl*.out'", cpNodeCoverageDir)
+		// Do not use single quotes here, as they will be interpreted literally.
+		// RunDockerExecCommand does not invoke a shell by default and it will split this
+		// string into a list of args.
+		cmd = fmt.Sprintf("find %s -maxdepth 1 -name antctl*.out", cpNodeCoverageDir)
 	} else {
 		cmd = fmt.Sprintf("find %s -maxdepth 1 -name 'antctl*.out'", cpNodeCoverageDir)
 	}
```

Comment:
Looks like we don't need to check `testOptions.providerName == "kind"` anymore?

Reply:
I thought so too originally, but notice the single quotes. If I include them for kind (docker exec), they will be interpreted literally (see my comment in the code). If I don't include them for the default case (ssh), the glob will be expanded by the shell (instead of the `find` command). There may be a unified command to achieve this, but I didn't want to spend too much time on this or risk breaking the non-kind case.

Reply:
Ok, got it, I thought the behaviors are the same.